server/settings.py

```diff
@@ -111,7 +111,11 @@ def env(variable, fallback_value=None):
     'spike:gc': {
         'task': 'apps.archive.content_purge',
         'schedule': crontab(minute=30)
-    }
+    },
+    'macros:print': {
+        'task': 'apps.macros.print_macros',
+        'schedule': timedelta(seconds=30),
```

Comment:
I think the delta should be bigger, ~10 minutes, as this is not going to change really often.